dataprep/utils.py

```diff
@@ -18,3 +24,43 @@ def is_notebook() -> Any:
         return False
     except (NameError, ImportError):
         return False
+
+
+def display_html(html_content: str) -> None:
+    """Writes HTML content to a file and displays in browser."""
+    if is_notebook():
+        display(HTML(html_content))
+    else:
+        file_path = LOADER.provider.module_path + "/" + LOADER.package_path + "/tmpf.html"
```

Comment:
Why is the temporary html file put under the package path? I think it would be better to put it under the temporary folder. You can get one using `tempfile.NamedTemporaryFile`.

Reply:
Right now it's stored in dataprep/dataprep/assets. Actually I think I should keep this file outside of the dataprep repo because anytime a developer uses the info function, it will change the tmpf.html file. So maybe I should move it just to whatever directory the user is in?

Reply:
> So maybe I should move it just to whatever directory the user is in?
You can use `tempfile.NamedTemporaryFile` for this purpose. It is better not to pollute the user's directory.

Reply:
So I tried to use this method before like:

with NamedTemporaryFile() ...:
    // code for writing to file 
    // open file in web browser

In this case, there's a line of code executed implicitly, once the code in the "with" block is finished executing. This code deletes the temp file. 

Sometimes the temp file is deleted before it can be opened by the browser though. One proposed solution was to add argument "delete = false" to NamedTemporaryFile. But this would defeat the purpose of having a temp file in the first place which is to delete files after they don't have a purpose anymore. 

This is why I implemented it this way. Let me know if this makes sense and if you still think anything should be changed!